delta/plugins/search/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/search/SearchRoutesSpec.scala

```diff
@@ -32,10 +32,11 @@ class SearchRoutesSpec extends BaseRouteSpec {
 
   private val fields = Json.obj("fields" := true)
 
-  private val publicProjects = Set(ProjectRef.unsafe("org", "project"), ProjectRef.unsafe("org2", "project2"))
-  private val suites         = Map(
+  private val publicProjects  = Set(ProjectRef.unsafe("org", "project"), ProjectRef.unsafe("org2", "project2"))
+  private val privateProjects = Set(ProjectRef.unsafe("org3", "project3"))
```

Comment:
Nitipicking - The name `privateProjects` is a bit confusing to me because in theory we could have a suite of one public project also (even though it doesn't make much sense) and the endpoint would return the same response as it would if the project was private.